cfy_manager/components/rabbitmq/rabbitmq.py

```diff
@@ -209,6 +205,10 @@ def _validate_rabbitmq_running(self):
     def _configure(self):
         systemd.configure(RABBITMQ,
                           user='rabbitmq', group='rabbitmq')
+        # note: this only runs if rabbitmq is being installed, so in case
+        # of an external rabbitmq, we'll keep the rabbitmq settings empty
```

Comment:
If this is what you want to do then the correct thing to use is a check of: if not self._install_manager():
  # Do the thing you want to do for external rabbitmq node

Reply:
nothing special to be done for an external rabbitmq node. The comment just notes that if we're on a manager node that is working with an external rabbitmq, this code won't run. I'll reword the comment to be actually legible.

What the code does is just defaults the rabbitmq-specific networks, to manager-wide networks, so that it defaults to the same IPs, but it's still possible to override the networks for just rabbitmq if the user so wishes